src/utils/cpm.js

```diff
@@ -2,9 +2,15 @@ import {auctionManager} from '../auctionManager.js';
 import {bidderSettings} from '../bidderSettings.js';
 import {logError} from '../utils.js';
 
-export function adjustCpm(cpm, bidResponse, bidRequest, {index = auctionManager.index, bs = bidderSettings} = {}) {
+export function adjustCpm(cpm, bidResponse, bidRequest, {index = auctionManager.index, bs = bidderSettings} = {}, gba = getBidAdjustment) {
```

Comment:
`gba` is a very thin wrapper around `bs`. it gets confusing if you can "decouple" them; if I pass in a particular instance of `bidderSettings`, I would expect it to be used throughout. to me this looks like you're asking for the same thing twice.